.php-cs-fixer.dist.php

```diff
@@ -62,12 +62,21 @@
         'numeric_literal_separator' => true,
         'phpdoc_order' => [
             'order' => [
+                'deprecated',
+                'final',
+                'readonly',
+                'internal',
+                'no-named-arguments',
```

Comment:
```suggestion
                'no-named-arguments',
```

this is "low-priority" tag IMO and I would move it before `@author`